frontend/lib/norent/homepage.tsx

```diff
@@ -298,24 +285,43 @@ export const NorentHomePage: React.FC<{}> = () => (
       </section>
 
       <section className="hero">
-        <div className="hero-body">
-          <div className="container jf-tight-container jf-has-text-centered-tablet jf-space-below-2rem">
-            <h2 className="title is-spaced">Legally vetted</h2>
-            <p className="subtitle is-size-5">
-              Our free letter builder was built with{" "}
-              <OutboundLink
-                href="https://www.justfix.nyc/about/partners"
-                rel="noopener noreferrer"
-              >
-                lawyers and non-profit tenants rights organizations
-              </OutboundLink>{" "}
-              across the nation to ensure that your letter gives you the most
-              protections based on your state.
-            </p>
+        <div className="hero-body jf-letter-preview-container">
+          <div className="container jf-has-text-centered-tablet">
+            <h3 className="is-size-5 is-spaced has-text-weight-normal">
+              Here’s a preview of what the letter looks like:
+            </h3>
             <br />
-          </div>
-          <div className="container jf-wide-container">
-            <LandingPagePartnerLogos />
+            <article className="message">
+              <div className="message-body has-background-grey-lighter has-text-left has-text-weight-light">
+                <p>Dear Landlord/Management.</p>
+                <br />
+                <p>
+                  I am writing to inform you that I have experienced a loss of
+                  income, increased expenses and/or other financial
+                  circumstances related to the pandemic. Until further notice, I
+                  will be unable to pay my rent due to the COVID-19 emergency.
+                </p>
+                <br />
+                <p>
+                  Tenants in Florida are protected from eviction for non-payment
+                  by Executive Order 20-94, issued by Governor Ron DeSantis on
+                  April 2, 2020.
+                </p>
+                <br />
+                <p>
+                  Tenants in covered properties are also protected from
+                  eviction, fees, penalties, and other charges related to
+                  non-payment by the CARES Act (Title IV, Sec. 4024) enacted by
+                  Congress on March 27, 2020.
+                </p>
+                <br />
+                <p>
+                  Along with my neighbors, I am organizing, encouraging, and/or
+                  participating in a tenant organization so that we may support
+                </p>
+              </div>
+              <div className="jf-letter-preview-fadeout" />
+            </article>
```

Comment:
Er, it would be ideal for us to reuse our `<NorentLetterContent>` component here so that we stay DRY and changes to the actual content mirror this, but this hard-coding is fine for now (unless you have objections to using the "real" letter content).

Can you file an issue w/ the NoRent milestone (I made it a few days ago) to make sure we address this later?

Reply:
Let's check in with Tahnee about it! It's possible that she might want the letter preview on the actual landing page to be a more formal tone like this, more as a descriptive element rather than actually showing a true sample of what the letter looks like (as with the actual embedded letter preview iFrame on the "The Letter" page).